views/elements/course-filters.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * Course Filter views
+ *
+ * A common filter element for all the backend pages
+ *
+ * @package Tutor\Views
+ * @subpackage Tutor\ViewElements
+ * @author Themeum <support@themeum.com>
+ * @link https://themeum.com
+ * @since 3.5.0
+ */
+
+use TUTOR\Input;
+
+if ( isset( $data ) ) : ?>
+<div class="tutor-admin-container tutor-admin-container-lg">
+	<div class="tutor-wp-dashboard-course-filter tutor-justify-<?php echo esc_attr( isset( $data['bulk_action'] ) && true === $data['bulk_action'] ? 'between' : 'end' ); ?>">
+		<?php if ( isset( $data['bulk_action'] ) && true === $data['bulk_action'] ) : ?>
+		<form id="tutor-admin-bulk-action-form" action method="post">
+			<input type="hidden" name="action" value="<?php echo esc_html( $data['ajax_action'] ); ?>" />
```

Comment:
For escaping the attribute value we need to use esc_attr
`<?php echo esc_attr( $data['ajax_action'] ); ?>`